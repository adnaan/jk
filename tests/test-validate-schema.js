import doTest from './validate-schema';

const jd = {
  firstName: 'John',
  lastName: 'Doe',
  age: 21,
};

doTest(jd);
