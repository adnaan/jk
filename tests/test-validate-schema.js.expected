Object:
ok
File:
ok
Module:
ok
